### PR TITLE
Fix backtick issues

### PR DESCRIPTION
**Pull request by imba-tjd**
After the first 4 commits, I used ```[^`]``[^`]``` to search and edited one by one locally.
These issues seem to have little similarity so I grouped them by path.
Some of my changes might be incorrect because I can't understand all of them.

### CODE REVIEW

File: docs/standard/data/xml/xslt-compiler-xsltc-exe.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ xsltc [options] [/class:<name>] <sourceFile> [[/class:<name>] <sourceFile>...]
 > [!NOTE]
 >  You must also include the compiled assembly as a reference in your application.  
   
- The xsltc.exe tool does not validate the class (`/class:``name`) or assembly (`/out:``assemblyName`) names. Errors are thrown by the common language runtime if the names are not valid.  
+ The xsltc.exe tool does not validate the class (`/class:` `name`) or assembly (`/out:` `assemblyName`) names. Errors are thrown by the common language runtime if the names are not valid.
```
Comment (rpetrusha) [inline docs/standard/data/xml/xslt-compiler-xsltc-exe.md]:
```suggestion
 The xsltc.exe tool does not validate the class (`/class:`*name*) or assembly (`/out:`*assemblyName*) names. Errors are thrown by the common language runtime if the names are not valid.  
```

File: docs/visual-basic/language-reference/objects/my-resources-object.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ Provides properties and classes for accessing the application's resources.
  You can access the application's culture-specific resource files from the `My.Resources` object. By default, the `My.Resources` object looks up resources from the resource file that matches the culture in the <xref:Microsoft.VisualBasic.ApplicationServices.ApplicationBase.UICulture%2A> property. However, you can override this behavior and specify a particular culture to use for the resources. For more information, see [Resources in Desktop Apps](../../../framework/resources/index.md).  
   
 ## Properties  
- The properties of the `My.Resources` object provide read-only access to your application's resources. To add or remove resources, use the **Project Designer**. You can access resources added through the **Project Designer** by using `My.Resources.``resourceName`.  
+ The properties of the `My.Resources` object provide read-only access to your application's resources. To add or remove resources, use the **Project Designer**. You can access resources added through the **Project Designer** by using `My.Resources`.`resourceName`.
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/objects/my-resources-object.md]:
```suggestion
 The properties of the `My.Resources` object provide read-only access to your application's resources. To add or remove resources, use the **Project Designer**. You can access resources added through the **Project Designer** by using `My.Resources.`*resourceName*.  
```

File: docs/visual-basic/language-reference/objects/my-resources-object.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ Provides properties and classes for accessing the application's resources.
...
   
 ## Properties  
- The properties of the `My.Resources` object provide read-only access to your application's resources. To add or remove resources, use the **Project Designer**. You can access resources added through the **Project Designer** by using `My.Resources.``resourceName`.  
+ The properties of the `My.Resources` object provide read-only access to your application's resources. To add or remove resources, use the **Project Designer**. You can access resources added through the **Project Designer** by using `My.Resources`.`resourceName`.  
   
- You can also add or remove resource files by selecting your project in **Solution Explorer** and clicking **Add New Item** or **Add Existing Item** from the **Project** menu. You can access resources added in this manner by using `My.Resources.``resourceFileName`.`resourceName`.  
+ You can also add or remove resource files by selecting your project in **Solution Explorer** and clicking **Add New Item** or **Add Existing Item** from the **Project** menu. You can access resources added in this manner by using `My.Resources`.`resourceFileName`.`resourceName`.
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/objects/my-resources-object.md]:
```suggestion
 You can also add or remove resource files by selecting your project in **Solution Explorer** and clicking **Add New Item** or **Add Existing Item** from the **Project** menu. You can access resources added in this manner by using `My.Resources.`*resourceFileName*`.`*resourceName*.  
```

File: docs/visual-basic/language-reference/xml-axis/xml-child-axis-property.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ object.<child>
...
 |`object`|Required. An <xref:System.Xml.Linq.XElement> object, an <xref:System.Xml.Linq.XDocument> object, a collection of <xref:System.Xml.Linq.XElement> objects, or a collection of <xref:System.Xml.Linq.XDocument> objects.|  
 |.<|Required. Denotes the start of a child axis property.|  
-|`child`|Required. Name of the child nodes to access, of the form [`prefix``:`]`name`.<br /><br /> -   `Prefix` - Optional. XML namespace prefix for the child node. Must be a global XML namespace defined with an `Imports` statement.<br />-   `Name` - Required. Local child node name. See [Names of Declared XML Elements and Attributes](../../../visual-basic/programming-guide/language-features/xml/names-of-declared-xml-elements-and-attributes.md).|  
+|`child`|Required. Name of the child nodes to access, of the form [`prefix:`]`name`.<br /><br /> -   `Prefix` - Optional. XML namespace prefix for the child node. Must be a global XML namespace defined with an `Imports` statement.<br />-   `Name` - Required. Local child node name. See [Names of Declared XML Elements and Attributes](../../../visual-basic/programming-guide/language-features/xml/names-of-declared-xml-elements-and-attributes.md).|
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/xml-axis/xml-child-axis-property.md]:
```suggestion
|`child`|Required. Name of the child nodes to access, of the form [`prefix:]name`.<br /><br /> -   `Prefix` - Optional. XML namespace prefix for the child node. Must be a global XML namespace defined with an `Imports` statement.<br />-   `Name` - Required. Local child node name. See [Names of Declared XML Elements and Attributes](../../../visual-basic/programming-guide/language-features/xml/names-of-declared-xml-elements-and-attributes.md).|  
```

File: docs/visual-basic/language-reference/xml-axis/xml-descendant-axis-property.md [hunk 5]
```diff
@@ -28,7 +28,7 @@ object...<descendant>
  Required. Denotes the start of a descendant axis property.  
   
  `descendant`  
- Required. Name of the descendant nodes to access, of the form [`prefix``:`]`name`.  
+ Required. Name of the descendant nodes to access, of the form [`prefix:`]`name`.
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/xml-axis/xml-descendant-axis-property.md]:
```suggestion
 Required. Name of the descendant nodes to access, of the form [`prefix:]name`.  
```

File: docs/visual-basic/programming-guide/program-structure/references-and-the-imports-statement.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ You can make external objects available to your project by choosing the **Add Re
   
  The `Imports` statement has the following syntax:  
   
- `Imports` [`|``Aliasname` =] `Namespace`  
+ `Imports` [`Aliasname` =] `Namespace`
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/program-structure/references-and-the-imports-statement.md]:
```suggestion
 `Imports [Aliasname =] Namespace`  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for correcting all of the formatting issues, @imba-tjd. I've left a number of suggested changes.

**Comment by imba-tjd** [comment]
I have accepted all of these changes, although in the middle I changed target branch and force-pushed once:joy:

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @imba-tjd. I'll merge your PR now.

**Pull request merged**